mashup.css

```diff
@@ -0,0 +1,98 @@
+* {
+	margin: 0;
+	padding: 0;
+	font-family: 'Gloria Hallelujah', cursive;
+}
+
+h1 {
+	text-align: center;
+	font-size: 50px;
+	font-family: 'Gloria Hallelujah', cursive;
+}
+
+h2 {
+	float: right;
+	padding: 5px 35px 15px 0;
+}
+
+#wrapper {
+	border: 4px solid black;
+	width: 970px;
+	height: 730px;
+	margin: 0 auto;
+	background-color: #5BB2BA;
+	border-radius: 7px;
+	font-family: 'Gloria Hallelujah', cursive;
+	position: relative;
+}
+
+p {
+	font-size: 30px;
+	white-space: pre;
+	margin: 30px 0 0 55px;
+	line-height: 4.5;
+}
+
+input {
+	font-size: 13px;
+	text-align: center;
+	display: block;
+    margin : 0 auto;
+	height: 19px;
+	text-transform: uppercase;
+	font-weight: bold;
+}
+
+input[type="submit"]{
+	visibility: hidden;
+}
+
+form.form {
+    height: 20px;
+}
+
+div.pic {
+	border: 3px solid black;
+	border-radius: 4px;
+	width: 180px;
+	height: 185px;
+	margin: 20px;
+}
+
+img {
+	margin: 10px 15px;
+}
+
+#firstPic, 
+#secondPic,
+#thirdPic,
+#fourthPic
+ {
+ 	position: absolute;
```

Comment:
I'm not a CSS expert, but there's probably a way to do this layout without having to absolutely position each of them. Take a look at [`vertical-align: middle`](https://developer.mozilla.org/en-US/docs/Web/CSS/vertical-align).